lang/python/python3-readline/Makefile

```diff
@@ -0,0 +1,31 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-readline
+PKG_VERSION:=8.0.0
+PKG_RELEASE:=$(AUTORELEASE)
+
+PYPI_NAME:=gnureadline
+PKG_HASH:=61eef72ed02dad415ede49752e972a1d2bd8c35c1e4464565d7effd806c99476
+
+PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
+PKG_LICENSE:=GPL-1.0-only
```

Comment:
```suggestion
PKG_LICENSE:=GPL-3.0-or-later
PKG_LICENSE_FILES:=LICENSE
```

Reply:
And not so sure about this either. Read my later comment.